packages/core/src/components/overflow-list/overflowList.tsx

```diff
@@ -170,8 +170,8 @@ export class OverflowList<T> extends React.PureComponent<IOverflowListProps<T>,
                 overflow: [],
                 visible: this.props.items,
             });
-        } else if (this.spacer.getBoundingClientRect().width < 1) {
-            // spacer has flex-shrink and width 1px so if it's any smaller then we know to shrink
+        } else if (this.spacer.getBoundingClientRect().width < 0.98) {
```

Comment:
should we be even more permissive here, maybe 0.5?

Reply:
that feels extreme? 0.9 may be sufficient